golem-worker-executor-base/src/worker.rs

```diff
@@ -1485,6 +1503,13 @@ impl RunningWorker {
                 while let Some(cmd) = receiver.recv().await {
                     waiting_for_command.store(false, Ordering::Release);
                     match cmd {
+                        WorkerCommand::ResumeReplay => {
+                            let mut store = store.lock().await;
+
+                            Ctx::resume_replay(&mut *store, &instance)
+                                .await
+                                .expect("resume_replay failed");
```

Comment:
We cannot panic here - this is a completely valid outcome (when trying to recovering a worker that failed in the past and will fail again).

The error of `resume_replay` must be handled exactly the same way as it is handled when it is coming from `prepare_instance` at the beginning of the invocation loop.